RELEASE.md

```diff
@@ -0,0 +1,118 @@
+# Releasing KubeArchive
+
+## Requirements/Assumptions
+
+* Using the release-notes tool that K8s uses
+* Using `kind/breaking`, `kind/documentation`, `kind/bug` and `kind/feature` labels
+* Using a code markdown block with release notes:
+    ```release-note
+    Something that will be added to the top of the release notes.
+    ```
+    or
+    ```release-note
+    NONE
+    ```
+* Chart version will be manually increased or will be the same as the application
+version to simplifiy the process.
+* No changelog, we can link to the branch comparison on GitHub, which will correspond
+to what we understand as changelog
+* Release notes are for users, we can delete the dependency information too, because it
+is too "developery".
+
+## Process
```

Comment:
Would this be a manual process when its time to release?

Reply:
No, it will be a manual GitHub workflow